buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy

```diff
@@ -573,6 +574,7 @@ class BeamModulePlugin implements Plugin<Project> {
         commons_csv                                 : "org.apache.commons:commons-csv:1.8",
         commons_io                                  : "commons-io:commons-io:2.7",
         commons_lang3                               : "org.apache.commons:commons-lang3:3.9",
+        commons_logging                             : "commons-logging:commons-logging:1.2",
```

Comment:
We explicitly remove commons_logging in flink with comments about jcl_over_slf4j. Do we need both?
https://github.com/apache/beam/blob/7d5d62e1ebf09eca0280bd7eea754a2223d0a416/runners/flink/job-server/flink_job_server.gradle#L75

Reply:
commons_logging is only used in the sdks/java/core tests, the flink override is about making sure we use slf4j